### ISSUE TITLE
Startup/autostart not working

### ISSUE THREAD

**Issue by CactiChameleon9** [vicr123/theshell#60]
I try to add an app to startup autostart, but after going through the menus, it does not apper on the list of autostart apps (empty), and does not work. I have tried adding a command as well, but the same result. Could it be a directory not existing issue?

**Comment by vicr123** [comment]
theShell follows the XDG autostart specification so if you look in `~/.config/autostart` you'll find all of the files. Try copying a desktop file from `/usr/share/applications` there and it should hopefully work 😅

As for the UI, I'm not too sure why that's broken. I'll need to look into it.

**Comment by CactiChameleon9** [comment]
Wow, you replied fast... Thanks! I will try what you suggested and get back to you

**Comment by CactiChameleon9** [comment]
I just made the .config/autostart folder, and then adding one from the ui worked, so you may want to add a mkdir command in your code somewhere, thanks for replying again!

**Comment by vicr123** [comment]
Gotcha. I'll just need to make sure the GUI creates the autostart folder if it's not already there. Thanks for letting me know :)

**Comment by CactiChameleon9** [comment]
Good, glad that I helped. Where should I make feature requests?

**Comment by CactiChameleon9** [comment]
should probably keep open until you add the code... :D. (Unless if you want to close it)

**Comment by vicr123** [comment]
Feature requests can be here too, but theShell is currently nearing EOL.

If you're still interested in a desktop environment from me, check out theDesk.

**Comment by CactiChameleon9** [comment]
The shell is near end of life? Really? Wow... Did you post about that on your website? I have briefly tried the desk - there wasnt much info about it, so I stayed with the shell.

1 Jul 2020 19:13:45 Victor Tran <notifications@github.com>:

Feature requests can be here too, but theShell is currently nearing EOL.

If you're still interested in a desktop environment from me, check out theDesk.

**Comment by vicr123** [comment]
I have not formally announced that theShell is EOL yet but once theDesk is released I will be encouraging everyone to shift over to theDesk.

I announce most of my software releases and general announcements on Discord.

**Comment by CactiChameleon9** [comment]
Yeah, that makes sense. I tend to stay away from discord myself for privacy concerns. Will shellOS be renamed? (Not that I use it, just interested)

1 Jul 2020 19:42:47 Victor Tran <notifications@github.com>:

I have not formally announced that theShell is EOL yet but once theDesk is released I will be encouraging everyone to shift over to theDesk.

I announce most of my software releases and general announcements on Discord.

**Comment by vicr123** [comment]
Most likely; stay tuned for updates 🙃

**Comment by CactiChameleon9** [comment]
Do you recommend I switch now, or should I wait until the offical release? (Very off topic :D)

1 Jul 2020 19:47:26 Victor Tran <notifications@github.com>:

Most likely; stay tuned for updates 🙃

**Comment by vicr123** [comment]
Your choice. You can also theoretically run it next to theShell if you wanted. :)
